docker.go

```diff
@@ -306,6 +307,31 @@ func (c *DockerContainer) Exec(ctx context.Context, cmd []string) (int, error) {
 	return exitCode, nil
 }
 
+func (c *DockerContainer) CopyToContainer(ctx context.Context, containerPath string, content io.Reader) error {
+	return c.provider.client.CopyToContainer(ctx, c.ID, containerPath, content, types.CopyToContainerOptions{})
```

Comment:
since I didn't need any special configuration, Im using the default types.CopyToContainerOptions, let me know if you think its better to already expose this

Reply:
Hi 👋 ! Is there a special need to expose this method, instead of directly calling the client provider method from the `CopyFileToContainer`? I feel a bit confused for having two methods in the API.

What do you think about this signature?
```golang
func (c *DockerContainer) CopyFileToContainer(ctx context.Context, hostPath string, containerPath string, fileMode int64) error
```
With this method, the client code must simply provide the source (localPath) and the target (containerPath), and the internals of the method will obtain all the information from those two params: will read the bytes and generate the Reader needed by the provider client. I think the API would be cleaner, but I'd like to hear your voice here.

BTW, I already implemented something like this in the past here: https://github.com/mdelapenya/lpn/blob/master/docker/docker.go#L141-L195

Reply:
@mdelapenya tks for your comment! The only reason I exposed this method is because it will allow clients to copy multiple files at once, since it exposes the io.Reader to an archive file. Im not really interested in this method to be honest, I'm using the one below that copies a single file.

What do you think? Should I set this first one to internal?

Reply:
Let's delegate to @gianarb for a decision. From my point of view, I'd go with the simplest use case, which is copying a file. We could build on top of this to copy multiples files later on, if/when needed. Sounds good?

Reply:
Sounds good to me! I removed the copyToContainer method. Leg me know what you guys think